backend/fms_core/viewsets/container.py

```diff
@@ -24,7 +22,7 @@
 )
 
 from ._utils import TemplateActionsMixin, TemplatePrefillsMixin, versions_detail
-from ._fetch_data import fetch_sample_data
+from .sample import SampleViewSet
```

Comment:
Is this import needed? It doesn't seem to be used anywhere.

Reply:
Nope it was part of an attempt to keep the defunct list_sample